hydrus/app.py

```diff
@@ -535,8 +694,10 @@ def app_factory(API_NAME: str="api") -> Flask:
                      "/<string:path>", endpoint="item_collection")
     api.add_resource(Item, "/" + API_NAME +
                      "/<string:path>/<int:id_>", endpoint="item")
-    api.add_resource(Items,"/"+ API_NAME +
-                     "/<string:path>/list",endpoint="items")
+    api.add_resource(Items, "/" + API_NAME +
+                     "/<string:path>/add/<int_list>", "/" + API_NAME +
+                     "/<string:path>/add", "/" + API_NAME +

```

Comment:
What will this path do?

Reply:
this basically allows us to redirect routes '/add' and '/add/1,2,3' to the same class